src/executor/graph_executor.cc

```diff
@@ -1551,16 +1550,16 @@ static nnvm::Symbol PartitionGraph(const nnvm::Symbol& src,
   std::vector<Context> aux_state_ctxes(aux_states.size());
 
   size_t i1 = 0, i2 = 0;
-  for (size_t i = 0; i < input_names.size(); ++i) {
-    if (i2 < aux_names.size() && aux_names[i2] == input_names[i]) {
+  for (const auto &input_name : input_names) {
```

Comment:
It should be:
```cpp
for (const auto& input_name : input_names) {
```

Reply:
Why is this a problem?

Reply:
Seems to be more consistent with the result of the file